cfgov/login/views.py

```diff
@@ -1,60 +1,8 @@
 from django.contrib import messages
-from django.contrib.auth import update_session_auth_hash
-from django.contrib.auth.decorators import login_required
-from django.http import Http404
-from django.shortcuts import redirect, render
+from django.shortcuts import redirect
+from django.urls import reverse_lazy
 
-from wagtail.admin.views import account
-from wagtail.admin.views.account import PasswordResetConfirmView
 
-from login.forms import CFGOVPasswordChangeForm, CFGOVSetPasswordForm
-
-
-# Override the Wagtail PasswordResetConfirmView to use our custom password
-# set form and remove lockouts when the form is valid.
-class CFGOVPasswordResetConfirmView(PasswordResetConfirmView):
-    form_class = CFGOVSetPasswordForm
-
-    def form_valid(self, form):
-        self.user.temporarylockout_set.all().delete()
-        return super().form_valid(form)
-
-
-# Overrided Django Views
-@login_required
-def change_password(request):
-    if not account.password_management_enabled():
-        raise Http404
-
-    user = request.user
-    can_change_password = user.has_usable_password()
-
-    if not can_change_password:
-        form = None
-
-    if request.POST:
-        form = CFGOVPasswordChangeForm(user=user, data=request.POST)
-
-        if form.is_valid():
-            form.save()
-            update_session_auth_hash(request, form.user)
-
-            messages.success(
-                request, "Your password has been changed successfully!"
-            )
-            return redirect("wagtailadmin_account")
-        else:
-            if "__all__" in form.errors:
-                for error in form.errors["__all__"]:
-                    messages.error(request, error)
-    else:
-        form = CFGOVPasswordChangeForm(user=request.user)
-
-    return render(
-        request,
-        "wagtailadmin/account/change_password.html",
-        {
-            "form": form,
-            "can_change_password": can_change_password,
-        },
-    )
+def lockout(request, credentials):
+    messages.error(request, "Account is locked.")
+    return redirect(reverse_lazy("wagtailadmin_login"))
```

Comment:
These lines are missing from the test coverage which fails the coverage check. I'm not sure there's an easy way to test the full django-axes path; maybe just a test that specifically invokes this view and confirms the redirect?

Reply:
Yeah, a test that the view redirects with the message should be good.

Reply:
In [11b5c24](https://github.com/cfpb/consumerfinance.gov/pull/7650/commits/11b5c24116974d2feb08174fcda35ace521d75eb)